server/superdesk/media/media_operations.py

```diff
@@ -119,7 +119,7 @@ def process_image(content, type):
 def crop_image(content, file_name, cropping_data):
     if cropping_data:
         file_ext = os.path.splitext(file_name)[1][1:]
-        if file_ext in ('JPG', 'jpg'):
+        if not file_ext or file_ext in ('JPG', 'jpg'):
```

Comment:
Tip: in general `set` is a better data structure for checks like these, because of its O(1) time for containment checking.

(but not that it really matters in this case with such a short tuple...)